documentation/puppetserver_vs_passenger.markdown

```diff
@@ -70,3 +70,22 @@ logged using `Puppet.info` will be logged in Puppet Server and would not have
 been under Passenger. See
 [Puppet Server Configuration Files: logback.xml](./config_file_logbackxml.md)
 for more information, as well as instructions on how to set log levels in Puppet Server.
+
+
+## External Command Execution
+
+When writing extensions that target the server, extension authors should be
+careful to use Puppet's built in execution helper: `Puppet::Util::Execution.execute`
+rather than native Ruby methods (eg "backticks", `system`, or `spawn`).
```

Comment:
`e.g.,`

might also help to show what a backtick looks like but I don't know if markdown is very friendly about that (``` doesn't work :D)